modules/wizard/view-wizard-page.php

```diff
@@ -32,9 +43,8 @@
 		<script>
 			var ajaxurl = '<?php echo esc_url( admin_url( 'admin-ajax.php', 'relative' ) ); ?>';
 		</script>
-		<?php do_action( 'admin_enqueue_scripts' ); ?>
```

Comment:
Why removing this line?

Reply:
This line has been moved to `PLL_Wizard::display_wizard_page()` to be able to document the local view variable `$styles` and because $this->styles is assigned on `PLL_Wizard::enqueue_scripts()`

See 
- https://github.com/polylang/polylang/blob/36ff6ffbec62dedc2e4dcfa6ed4ce96554c9927c/modules/wizard/wizard.php#L279
- https://github.com/polylang/polylang/blob/36ff6ffbec62dedc2e4dcfa6ed4ce96554c9927c/modules/wizard/wizard.php#L282
- https://github.com/polylang/polylang/blob/36ff6ffbec62dedc2e4dcfa6ed4ce96554c9927c/modules/wizard/view-wizard-page.php#L19